src/FSharp.Formatting.Common/Templating.fs

```diff
@@ -48,10 +48,15 @@ type FrontMatterFile =
                 // Allow empty lines in frontmatter
                 let isBlankLine = String.IsNullOrWhiteSpace line
                 isBlankLine || line.Contains(":"))
-            |> Seq.filter (String.IsNullOrWhiteSpace >> not)
-            |> Seq.map (fun line ->
-                let parts = line.Split(":")
-                parts.[0].ToLowerInvariant(), parts.[1])
+            |> Seq.choose (fun line ->
+                let parts = line.Split(":") |> Array.toList
```

Comment:
This doesn't seem safe when `line` is null.

Reply:
Good catch. I moved this within the if check